packages/web-app-files/src/components/SideBar/Shares/FileShares.vue

```diff
@@ -142,7 +142,7 @@ export default defineComponent({
   },
   computed: {
     ...mapGetters(['configuration']),
-    ...mapGetters('runtime/spaces', ['spaceMembers']),
+    ...mapGetters('runtime/spaces', ['spaceMembers', 'spaces']),
```

Comment:
`this.spaces` is used in `matchingSpace()` and we were not sure about removing it completely, it seemed the mapped getter here might be missing